.github/ISSUE_TEMPLATE/issue-dev-frontend.md

```diff
@@ -0,0 +1,52 @@
+---
+name: 💻 Fronted dev team internal issue template
+about: Internal issue template for Frontend dev team
+title: ""
+labels: ""
+assignees: ""
+---
+
+<!--
+  Checking this will indicate that the current issue is still draft
+-->
+
+- [x] **Draft**
+
+## Linked Issue
+
+<!-- !REQUIRED -->
+<!--
+  Provide a link to related Issue or Pull requests
+  check example for usage reference
+  example:
+  - #1451
+  - https://github.com/iCloudHospital/iCloudHospital/issues/1120
+  - [ ] #1451
+  - [ ] https://github.com/iCloudHospital/saas-next/issues/7
+-->
+
```

Comment:
@henrynoowah 

I thought we decided to add `<br>` tag between headings

Reply:
@froggy1014 
Decided to not add br and developer to add br accordingly